src/scss/panel.scss

```diff
@@ -15,11 +15,11 @@
 
 /**
  * Apply to an aside header block
- * @requires {mixin} oTypographyHeading2
+ * @requires {mixin} oTypographyDisplay
  * @requires {function} oColorsGetColorFor
  */
 @mixin oAsidePanelHeader {
-	@include oTypographyHeading2;
+	@include oTypographyDisplay($scale: 3, $line-height: 31px);
```

Comment:
You _can_ use `oTypographyHeadingLevel2` here, but not sure whether you already discussed this with design/adam :)